torch/autograd/__init__.py

```diff
@@ -477,6 +475,7 @@ def _is_checkpoint_valid():
 
 
 def variable(*args, **kwargs):
+    """Provide runtime error for deprecated autograd variable."""
```

Comment:
Could we turn this into a skip instead?

Reply:
@soulitzer sure, that sounds good. Thanks!